PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+
+**Name**: *Small scale solar self-consumption*
+
+**Developed by**: *[PeerCo](www.peerco.earth)*
+
+**Revision date**: *July 5th 2024*
+
+**Sectoral scope**:  *Energy / Energy Distribution / Energy Demand*
+
+
+## Summary description
+
+This methodology for measurements and verification of self-consumed small-scale on-site solar generation on a given site. The purpose of this methodology is to generate *Environmental Attribute Certificates* representing the self-consumed solar energy (at a specific time and geographical location), which can then be used in reporting or traded.
+
+## Definitions
+
+This section provides definitions of the key aspects of this methodology.
+
+### Normative language
+
+The following terminology is used throughout this document to outline requirements and guidelines (accordingly with common standardisation practices):
+- **shall** indicates a *requirement*,
+- **should** indicates a *recommendation*,
+- **may** indicates an *option* that is permitted.
+
+### Acronyms
+
+*(this section shall contains the list of acronyms in alphabetical order)*
+
+| Acronym | Full text           |
+|---------|---------------------|
+| CDM     | Clean development mechanism |
+| EAC     | Energy Attribute Certificate |
+| GHG     | Greenhouse gas(es)  |
+| PV      | Photovoltaic        |
+| UN      | United Nations      |
+
+### Glossary
+
+- **Digital certificate** is a generic term designating environmental attributes in digital form generated using this methodology. Examples of these include EACs and carbon credits. 
+
+- **Emission intensity** (or *carbon intensity*) is a measure of how clean grid electricity is. It refers to how many grams of carbon dioxide equivalent (gCO2e) are released to produce a kilowatt hour (kWh) of electricity. In this methodology, an *average* measure of this factor *on the demand side* is considered at every hour and for a given grid area, meaning that this number reflects the carbon intensity of the *average kilowatt-hour consumed* in this given hour and grid region.
+
+- **High resolution data** is data whose time resolution is hourly or sub-hourly (e.g. every 15 or 30 minutes).
+
+- **Grid region** is a generic term used here to refer to a specific sub-division of a large-scale power system. This may be a balancing region, a country-wide grid, a market-bidding region depending on the local context. The spatial resolution used for the grid region used in a given application should be as granular as allowed by the data available.
+
+- **PV asset** is a set of photovoltaic arrays and associated inverter(s) converting solar irradiance to electrical power.   
+
+## Applicability conditions
+
+This section lays out the requirements on a given context for this methodology to be applicable.
+
+### System eligibility requirements
+
+The system to which the methodology is applied shall meet the following requirements to be eligible:
+
+1. the rating of the PV is between 30 kWp and 5 MWp
```

Comment:
Add kWp and MWp to the acronym table?

Reply:
We had considered it obvious in our context, but it's potentially good to remind in an international context. This is now added [here](https://github.com/wattcarbon/WEATS/pull/2/commits/2731bd57453f447cecf3fa147aeb66bbd7c8b8e7)

Reply:
@Pierre-VF what is the reason for these sizing constraints? Is this purposely meant to exclude residential solar?

Reply:
We're actually removing the lower bar (we want small scale to happen). However, the upper bar is basically to explicitly exclude large-scale